src/NServiceBus.Core/obsoletes-v8.cs

```diff
@@ -349,9 +349,10 @@ namespace NServiceBus
 {
     using System;
 
+    // The type itself can't be configured with TreatAsErrorFromVersion 8 as downstream extension methods require the type to obsolete their own extension methods.
     [ObsoleteEx(
         Message = "Configure the transport via the TransportDefinition instance's properties",
-        TreatAsErrorFromVersion = "8.0",
+        TreatAsErrorFromVersion = "9.0",
         RemoveInVersion = "9.0")]
```

Comment:
Should this, and others too, be `RemoveInVersion="10.0"`?
Otherwise, the generated error message is a bit confusing.

Reply:
I'm not sure it's worthwile keeping them around for more than one major. What will the behavior be if we remove `TreatAsErrorFromVersion ` completely? will it still be just a warning at v8?